docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: astropy
+
+.. _environment_variables:
+
+*********************
+Environment variables
+*********************
+
+
+Runtime
+-------
+
+.. glossary::
+
+    ``XDG_CONFIG_HOME``
```

Comment:
Both entries should document what happens if the variables are not set or the directories they point to don't exist.